packages/generator/src/helper.ts

```diff
@@ -294,16 +302,25 @@ const drawInputByTextSchema = async (arg: {
         pdfFontValue.widthOfTextAtSize(testString, size) + (testString.length - 1) * characterSpacing;
       return width <= testStringWidth;
     };
-    const splitedLines = getSplittedLines(inputLine, isOverEval);
-    const drawLine = (splitedLine: string, splitedLineIndex: number) => {
+    const splitLines = getSplittedLines(inputLine, isOverEval);
+
+    const drawLine = (line: string, lineIndex: number) => {
       const textWidth =
-        pdfFontValue.widthOfTextAtSize(splitedLine, size) +
-        (splitedLine.length - 1) * characterSpacing;
-      page.drawText(splitedLine, {
+        pdfFontValue.widthOfTextAtSize(line, size) + (line.length - 1) * characterSpacing;
+
+      const fontkitFont = fontkit.create(
+        Buffer.from(font[templateSchema.fontName as string].data as ArrayBuffer)
```

Comment:
templateSchema.fontName is not string, it can be undefined.
if templateSchema.fontName is undefined, we should use a fallback font.

```
      const fontkitFont = fontkit.create(
        Buffer.from(font[templateSchema.fontName ?? fallbackFontName].data as ArrayBuffer)
      );
```